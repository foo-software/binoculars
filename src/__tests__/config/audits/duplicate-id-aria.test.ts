import auditDuplicateIdAria from '../../../config/audits/duplicate-id-aria';

describe('duplicate-id-aria audit', () => {
  it('should match snapshot', () => {
    expect(auditDuplicateIdAria('en')).toMatchSnapshot();
  });
});
