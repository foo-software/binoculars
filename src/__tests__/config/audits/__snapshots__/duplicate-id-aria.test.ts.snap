// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`duplicate-id-aria audit should match snapshot 1`] = `
Object {
  "description": "The value of an ARIA ID must be unique to prevent other instances from being overlooked by assistive technologies. [Learn more](https://web.dev/duplicate-id-aria/).",
  "title": "ARIA IDs are unique",
}
`;
